src/utils/index.js

```diff
@@ -2,7 +2,6 @@
 /** @typedef {import("../index").farmhand.item} farmhand.item */
 /** @typedef {import("../index").farmhand.plotContent} farmhand.plotContent */
 /** @typedef {import("../index").farmhand.shoveledPlot} farmhand.shoveledPlot */
-/** @typedef {import("../index").farmhand.cropTimetable} farmhand.cropTimetable */
```

Comment:
I think we'll want to keep this but change it to:

```ts
/** @typedef {import("../index").farmhand.cropTimeline} farmhand.cropTimeline */
```

Reply:
oh good call. i originally removed it because it's not a "special" type any more but i see now where it's being referenced and complaining. added back!

Reply:
I actually had this slightly wrong as `farmhand.cropTimeline` is not defined as a type. I've fixed this up in https://github.com/jeremyckahn/farmhand/commit/9a9135d55d35e175567ddd14fc7663ed07a3a4f8.